Index: tests/TextMeTest.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- tests/TextMeTest.php	(revision 57ac95cd47a90b35e7c40896ed467fdae4fb3dce)
+++ tests/TextMeTest.php	(date 1596990752729)
@@ -37,5 +37,6 @@
         yield ['Foo_Bar'];
         yield ['foo_BAR'];
         yield ['fOo_BAR'];
+        yield ['fOO_BAR'];
     }
 }
